### ISSUE TITLE
Improve the trusted_proxies descriptions in the wiki ?

### ISSUE THREAD

**Issue by leo15dev** [AdguardTeam/AdGuardHome#3382]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Issue Details

* **Version of AdGuard Home server:**
  * v0.107.0-a.111+2f661df8
* **How did you install AdGuard Home:**
  *  Docker
* **How did you setup DNS configuration:**
  *  loT
* **If it's a router or IoT, please write device model:**
  *  Raspberry Pi 4B
* **CPU architecture:**
  *  ARM
* **Operating system and version:**
  *  Ubuntu 20.04.2 LTS

### Expected Behavior

I used cloudflare argo tunnel, Nginx reverse proxy (DoH), the Adguard Home build with docker (custom bridge network, e.g. the address is 172.20.0.1), everything works fine until updated to  v0.107.0-a.111+2f661df8 .
v0.107.0-a.111+2f661df8 added trusted_proxies feature, it should be work fine without needing user to modify any configuration.

### Actual Behavior

After updating to v0.107.0-a.111+2f661df8, the DoH started to refuse any DNS query.  Until I added 172.20.0.1/24 in the trusted_proxies of AdGuardHome.yaml. Finally, it started work as usual.  I think it belongs to the document improvement. Or  in the change log, it should  reminder user to add such configuration in the  AdGuardHome.yaml. So I reported the issue here. Thank you.

### Screenshots

<details><summary>Screenshot:</summary>

</details>

**Comment by ainar-g** [comment]
Thanks for the report.  We'll add a separate section about this in the docs of the next beta as well as the final v0.107.0 release.

**Comment by ainar-g** [comment]
@leo15dev, just a head-up.  The breaking behaviour was a mistake, and v0.107.0-b.7 reverts to the old behaviour instead.  We'll still cover this change in the release notes, obviously, but there is less to cover now, so to say.

**Comment by leo15dev** [comment]
@ainar-g Thanks for your information. I did the test on the version v0.107.0-a.121+d0772fda. If I do not  add 172.20.0.1/24 in the trusted_proxies of AdGuardHome.yaml,  the DoH still work as usual, however, the client IP showed in the Query Log page is the 172.20.0.1. But, if I added 172.20.0.1/24 in the trusted_proxies of AdGuardHome.yaml, the client IP would show the its real IP. I think it was conformed to the "Trusted proxies" description in  [AdGuard Home v0.107.0-b.6 ](https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.0-b.6). Thank you!
